Sources/Extensions/SwiftStdlib/StringExtensions.swift

```diff
@@ -620,10 +621,10 @@ public extension String {
 	///   - length: amount of characters to be sliced after given index.
 	/// - Returns: sliced substring of length number of characters (if applicable) (example: "Hello World".slicing(from: 6, length: 5) -> "World")
 	public func slicing(from i: Int, length: Int) -> String? {
```

Comment:
Now we can subscript because it's a collection so this can probably go too 😅

Reply:
I don't know about this one. I think handles some safe convenience cases that maybe aren't done with the subscript. @SD10  @omaralbeik  What you think?

Reply:
I think we should turn it into a safe subscript that accepts a `Range<String.Index>` and returns a `Substring?` then. The name of this method doesn't make sense, I'd expect to get an `ArraySlice` out of it. Since it has `ing` it also makes me think it's mutating and it's not 😅

Reply:
I don't know if it can be turned in a subscript with range because the second parameter is a length, not an end index 😅 I think worth just rename and returning a substring instead. @SD10 @omaralbeik  What you think?

Reply:
That's a good point @LucianoPAlmeida 🤔 But I'm having trouble thinking of a place where someone would be using a length and not an `endIndex` considering they have a `startIndex`. You can always get the `endIndex` by incrementing the `startIndex` by `n`

Reply:
I agree so it will be better deprecating this?

Reply:
Yeah, we can always replace it with the `subscript` version in another PR. Sorry to make so many changes. I should've thought about this before we even released SwifterSwift 4.0. It completely slipped my mind

Reply:
What about the mutating versions? Should be maintained?

Reply:
@LucianoPAlmeida lets not make this change right now... I just opened the file and there are a ton of these `slice` methods. We also already have `subscripts` from ranges but they return `String?` and not `Substring?` It's too much for me to look at right now...

Reply:
@SD10 No problem we can open a issue/PR later with only this methods to discuss :)